src/api/app/views/webui/projects/maintained_projects/_create_dialog.html.haml

```diff
@@ -0,0 +1,19 @@
+.modal.fade#new-maintenance-project-modal{ tabindex: '-1', role: 'dialog', aria: { labelledby: 'new-maintenance-project-modal', hidden: true } }
+  .modal-dialog.modal-lg{ role: 'document' }
+    .modal-content
+      = form_tag nil, method: :create do
+        .modal-header
+          %h5.modal-title#new-maintenance-project-modal
+            Add New maintained project to #{project_name}
```

Comment:
`New` isn't needed, so something like `Add Project to Maintain in #{project_name}"` reads much better.